stalker/code/objects/items/artifacts/artifacts.dm

```diff
@@ -261,7 +262,101 @@
 		mob.adjustBruteLoss(-0.5)
 	return 1
 
-	//ПОЯС
+/obj/item/artifact/thorn
+	name = "thorn"
+	desc = "The result of the interaction between the anomaly Burnt Fuzz and the body of a careless stalker. The Thorn artifact pokes the body of its owner, no matter what. But it also helps clean the body of radionucliodes. Quite widespread and cheap."
+	eng_desc = "The result of the interaction between the anomaly Burnt Fuzz and the body of a careless stalker. The Thorn artifact pokes the body of its owner, no matter what. But it also helps clean the body of radionucliodes. Quite widespread and cheap."
+	icon_state = "thorn"
+	radiation = -1
+	level_s = 1
+
+/obj/item/artifact/thorn/Think(mob/user)
+	if(!..()) return 0
+	if(istype(user, /mob/living/carbon))
+		var/mob/living/carbon/mob = user
+		mob.bleed(1)
+	return 1
+
+/obj/item/artifact/urchin
+	name = "urchin"
+	desc = "The anomaly Burnt Fuzz very rarely gives rise to this artifact. Blood pressure rises, the body gets rid of a large amount of red blood cells. But along with them the stored radiation leaves the body as well. In his fundamental work titled \"Ionization and polarization of the components of rare artifacts\", Sakharov noted that the content of this formation has a critical stability, and it's not realistic to create such an artifact in lab conditions in the next ten years."
+	eng_desc = "The anomaly Burnt Fuzz very rarely gives rise to this artifact. Blood pressure rises, the body gets rid of a large amount of red blood cells. But along with them the stored radiation leaves the body as well. In his fundamental work titled \"Ionization and polarization of the components of rare artifacts\", Sakharov noted that the content of this formation has a critical stability, and it's not realistic to create such an artifact in lab conditions in the next ten years."
+	icon_state = "urchin"
+	radiation = -3
+	level_s = 4
+
+/obj/item/artifact/urchin/Think(mob/user)
+	if(!..()) return 0
+	if(istype(user, /mob/living/carbon))
+		var/mob/living/carbon/mob = user
+		mob.bleed(1)
+	return 1
+
+/obj/item/artifact/slug
+	name = "slug"
+	desc = "Formed by the \"Fruit Punch\" anomaly. The negative qualities of this artifact are compensated by the fact that it heightens the coagulation quality of blood. It's not often that one runs into such an artifact, and they pay well for it too."
+	eng_desc = "Formed by the \"Fruit Punch\" anomaly. The negative qualities of this artifact are compensated by the fact that it heightens the coagulation quality of blood. It's not often that one runs into such an artifact, and they pay well for it too."
+	icon_state = "slug"
+	art_armor = list(laser = -10, bio = -10)
+	radiation = 0
+	level_s = 2
+
+/obj/item/artifact/slug/Think(mob/user)
+	if(!..()) return 0
+	if(istype(user, /mob/living/carbon/human))
+		var/mob/living/carbon/human/H = user
+		if(!H.bleedsuppress) //so you can't stack bleed suppression
+			H.suppress_bloodloss(0.20)
```

Comment:
New artefact effects will be added soon as 'prefabs', so you can pull from there and make them similar to the game. For now, this works.